examples/coordinates/plot_sgr-coordinate-frame.py

```diff
@@ -205,13 +171,15 @@ def sgr_to_galactic(sgr_coord, gal_frame):
 ##############################################################################
 # Plot the points in both coordinate systems:
 
-fig,axes = plt.subplots(2, 1, figsize=(6,12),
-                        subplot_kw={'projection': 'aitoff'})
+fig, axes = plt.subplots(2, 1, figsize=(8, 10),
```

Comment:
Now that we have `WCSAxes` inside astropy, should it be used here? It seems a perfect place for a nice example...
(But don't do it if it takes too much time.)